99.wc_object/wc.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Wc
+  attr_reader :line_count, :word_count, :byte_count, :name
+
+  def initialize(text:, filename: '')
```

Comment:
```suggestion
  def initialize(text, filename = '')
```

キーワード引数にしなくてもいいかもしれませんね
https://bootcamp.fjord.jp/pages/dont-abuse-keyword-args

Reply:
`files.map { |file| Wc.new(file.read, file.path) }`
そうですね、引数に十分情報があるので不要でした。